tests/async_fixtures/test_async_fixtures.py

```diff
@@ -37,3 +37,14 @@ async def async_fixture_method(self):
     @pytest.mark.asyncio
     async def test_async_fixture_method(self):
         assert self.is_same_instance
+
+
+@pytest.fixture()
```

Comment:
The test file you found uses the "old" way of writing tests directly in a module. In the past, these kinds of tests caused side effects (e.g. failing to clean up resources resulting in a ResourceWarning) that only appeared much later in the test suite. Therefore, we moved to using [Pytester](https://docs.pytest.org/en/stable/how-to/writing_plugins.html#testing-plugins) for all pytest-asyncio tests.

I'd appreciate, if you rewrote the test to use *Pytester,* even though it's mostly boilerplate. You can check the the file `test_function_scope.py` for an example.

Although it's not strictly necessary, I suggest to use strict mode in those tests (or both, if there's different behavior depending on the mode).